NicolaeChristof/Descend-Into-Darkness
# Feature elevator

Here is the PR for changes to the Mine level. In this PR I added the elevator to the mine level, the elevator first has to be powered on then you can use it with E. I also added lights to the environment. I also fixed a bug with the skeleton enemy, now you need to place two target points in the scene and reference them from the skeleton in question.

How to test:
- open the mine level and check the lighting, it should only contain a bunch of spot lights
- the skeleton should patrol properly now
- the elevator must be turned on to work, approach the red panel and press E
- go into the elevator and press E on the control panel (right wall), the elevator should slowly take you down to a lower level, upon pressing the button again it should take you to the bottom level, if pressed again it will go back up to the previous level

No Known Bugs
